go.mod

```diff
@@ -4,6 +4,7 @@ go 1.19
 
 require (
 	entgo.io/ent v0.12.2
+	github.com/Junzki/link-preview v0.0.0-20190616153136-4d94cb3f9ebf
```

Comment:
This package had a very ugly implementation and didn't quite follow the regulations and best practices in Go. 

However, I think such logic is simple enough to wrap one by ourselves. I will make one for you.

Reply:
Didn't find any well-maintained packages, though.

Reply:
It's ok. Thank you for doing the research!